src/main/java/com/easypost/model/beta/EndShipper.java

```diff
@@ -0,0 +1,314 @@
+package com.easypost.model.beta;
+
+import com.easypost.EasyPost;
+import com.easypost.exception.EasyPostException;
+import com.easypost.net.EasyPostResource;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public final class EndShipper extends EasyPostResource {
```

Comment:
I thought we were going to extend the Address class, and then simply override the address methods, since ultimately an `EndShipper` is really just an `Address`?

Reply:
Yes, we can do that. But I don't want users to access other functions from the `Address` class such as `create and verify()`

Reply:
You can override those functions and simply throw a NotImplemented exception

Reply:
Personally, I don't like overriding every method(except getters and setters) from the parent class `Address`.

Reply:
Per our convo, let's decouple the EndShipper object from the Address object (do not extend). This has been corrected in Ruby.